### PR TITLE
Corrected Indentation

### PR DESCRIPTION
**Pull request by manvantarah**

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -30,12 +30,12 @@ kind: KadaluStorage
...
-    spec:
-        type: Replica1
-        storage:
-        - node: kube1      # node name as shown in `kubectl get nodes`
-          device: /dev/vdc # Device to provide storage to all PVs
-```
+spec:
+    type: Replica1
+    storage:
+      - node: minikube      # node name as shown in `kubectl get nodes`
```
Comment (amarts) [inline docs/quick-start.md]: lets keep this same as 'kube1'.

File: docs/quick-start.md [hunk 2]
```diff
@@ -30,12 +30,12 @@ kind: KadaluStorage
...
+        device: /root/disk  # Device to provide storage to all PVs
```
Comment (amarts) [inline docs/quick-start.md]: let this also be '/dev/vdc'

**Review by amarts — COMMENTED** [review]
Only fix indentation, not the content

File: docs/quick-start.md (hunk 1 above)
Comment (aravindavk) [inline docs/quick-start.md]: change back to kube1, and actual device as earlier. Minikube example is not prefered for the website

**Review by aravindavk — APPROVED** [review]

**Pull request merged**